Add sample qa instructions (#70)
elastisys/compliantkubernetes: docs/operator-manual/qa.md

> # Quality assurence

Reviewer: ```suggestion
# Quality Assurance
```

> When you have created your Compliantkubernetes cluster it can be wise to run some checks to ensure that it works as expected.

Reviewer: ```suggestion
When you have created your Compliant Kubernetes cluster it can be wise to run some checks to ensure that it works as expected.
```

This document details some snippets that you can follow in order to ensure some functionality of the cluster.

> ### Customer API and Harbor access

Reviewer: ```suggestion
## Customer API and Harbor access
```

> 1. Pre-requisites.

Reviewer: Please use heading 3. "###" instead of itemize. I now realize that itemize is really tedious from an indenting perspective.

    - You've got Docker installed.
    - You've exported `CK8S_CONFIG_PATH` in your shell.
    - You've set `baseDomain` in your shell to what's used in your cluster.

>     - Your current working directory is at the `compliantkubernets-apps` repository.

Reviewer: ```suggestion
    - Your current working directory is the `compliantkubernets-apps` repository.
```

    - You've installed the `kubectl` plugin `kubelogin`.
        See [instructions](https://github.com/int128/kubelogin#setup) on how to install it.

2. Create and set user kubeconfig.

>     ```shellSession

Reviewer 2: ```suggestion
    ```shellSession
    bin/ck8s user-kubeconfig
```

>     pushd ${CK8S_CONFIG_PATH} > /dev/null || exit 1

Reviewer: Wouldn't this kill my shell?

>     sops -d -i user/kubeconfig.yaml

Reviewer: I get `Error: cannot operate on non-existent file`.

```
cklein@minitrue:~/.ck8s/exoscale$ pwd
/home/cklein/.ck8s/exoscale
cklein@minitrue:~/.ck8s/exoscale$ ls -l
total 52
-rw-rw-r-- 1 cklein cklein    66 feb 23 18:54 blah.sh
drwxrwxr-x 4 cklein cklein  4096 feb 23 13:02 cksc-config
drwxrwxr-x 4 cklein cklein  4096 feb 23 11:52 ckwc0-config
-rw-rw-r-- 1 cklein cklein 14030 feb 23 20:18 sc-config.yaml
-rw-rw-r-- 1 cklein cklein  8010 feb 24 12:14 secrets.yaml
drwxrwxr-x 2 cklein cklein  4096 feb 23 13:28 user
-rw-rw-r-- 1 cklein cklein 10630 feb 24 14:59 wc-config.yaml
cklein@minitrue:~/.ck8s/exoscale$ ls -l user/
total 0
```

>     export KUBECONFIG=${pwd}/user/kubeconfig.yaml

Reviewer: ```suggestion
    export KUBECONFIG=$(pwd)/user/kubeconfig.yaml
```

>     popd > /dev/null || exit 1

Reviewer: Same as above. Wouldn't this make my shell exit?
